.github/workflows/rebase-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: Rebase a pr
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  rebase_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@master
```

Comment:
Does this mean that it rebases PRs on master? Could we make ut so that it rebases against the base branch (ir is it too complicated)? If not, tgen this shoyld be part of the name and/or the label to make it clear to users.

Reply:
Yeah, sorry, I figured it out at some point during the review, but forgot to remove this comment 😇